backend/coins/btc/transactions/transactions.go

```diff
@@ -394,11 +394,16 @@ func (transactions *Transactions) Balance() *accounts.Balance {
 }
 
 // byHeight defines the methods needed to satisify sort.Interface to sort transactions by their
-// height. Special case for unconfirmed transactions (height <=0), which come last.
+// height. Special case for unconfirmed transactions (height <=0), which come last. If the height
+// is the same for two txs, they are sorted by the created (first seen) time instead.
 type byHeight []*TxInfo
 
 func (s byHeight) Len() int { return len(s) }
 func (s byHeight) Less(i, j int) bool {
+	// Secondary sort by the time we've first seen the tx in the app.
+	if s[i].Height == s[j].Height && s[i].createdTimestamp != nil && s[j].createdTimestamp != nil {
+		return s[i].createdTimestamp.Before(*s[j].createdTimestamp)
```

Comment:
Wouldn't you want to sort by `createdTimestamp` even if `s[x].Height <=0 `?

Reply:
It is, but as a secondary sort. Primary sort is still the block height.